src/Maestro/Maestro.Web/Controllers/DependenciesController.cs

```diff
@@ -0,0 +1,44 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.HealthMetrics;
+using Microsoft.Extensions.Logging;
+
+namespace Maestro.Web.Controllers
+{
+    [Route("[controller]")]
+    [Route("_/[controller]")]
+    public class DependenciesController : ControllerBase
+    {
+        private readonly IRemoteFactory remoteFactory;
+        private readonly ILogger<DependenciesController> logger;
+
+        public DependenciesController(IRemoteFactory factory, ILogger<DependenciesController> logger)
+        {
+            remoteFactory = factory;
+            this.logger = logger;
+        }
+
+        [HttpGet("getSubscriptionDependencyDetails/{host}/{account}/{project}/{*branch}")]
```

Comment:
the "get" doesn't need to be repeated here (it's in the verb).  "SubscriptionDependencyDetails" is a pretty vague name.  It definitely needs a big ol' pile of comments explaining what it's doing. It'll also need the [SwaggerResponse] attribut that the other controllers have to describe what the return object looks like and what status codes it will return.

Reply:
Does putting it in the verb change the path string? I'd like to have clarity in the code making the call as much as possible. What name would you prefer? I'll add the attribute.

Reply:
I honestly have no idea what this code is doing, so I can't comment on a good name.  Though that definitely means the current name probably isn't a good what.  I don't know what a "subscription depedendency detail" is, or why I'd want to get one.

REST URL's almost never have verbs in them, since you are "getting" some resource. The path is the name of the resource you want to get, the verb is... well... the verb.